GOVERNANCE.md

```diff
@@ -128,10 +135,71 @@ the issue tracker is:
   either the proposal is dropped or the objecting members are persuaded. If
   there is an extended impasse, a motion for a vote may be made.
 
+## Collaborator Nominations
+
+Any existing Collaborator can nominate an individual making significant
+and valuable contributions across the Node.js organization to become a new
+Collaborator.
+
+To nominate a new collaborator, open an issue in the [nodejs/node][]
+repository, with a summary of the nominee's contributions, including but
+not limited to:
+
+* Commits in the [nodejs/node][] repository.
+  * Can be shown using the link
+    `https://github.com/nodejs/node/commits?author=${GITHUB_ID}`
+    (replace `${GITHUB_ID}` with the nominee's GitHub ID).
+* Pull requests and issues opened in the [nodejs/node][] repository.
+  * Can be shown using the link
+    `https://github.com/nodejs/node/pulls?q=author%3A${GITHUB_ID}+`
+* Comments and reviews on issues and pull requests in the
+  [nodejs/node][] repository
+  * Can be shown using the links
+    `https://github.com/nodejs/node/pulls?q=reviewed-by%3A${GITHUB_ID}+`
+    and `https://github.com/nodejs/node/pulls?q=commenter%3A${GITHUB_ID}+`
+* Assistance provided to end users and novice contributors
+* Participation in other projects, teams, and working groups of the
+  Node.js organization
+  * Can be shown using the links
+    `https://github.com/search?q=author%3A${GITHUB_ID}++org%3Anodejs&type=Issues`
+    and
+    `https://github.com/search?q=commenter%3A${GITHUB_ID}++org%3Anodejs&type=Issues`
+* Other participation in the wider Node.js community
+
+Mention @nodejs/tsc and @nodejs/collaborators in the issue to notify the TSC
+and other collaborators about the nomination.
```

Comment:
Since TSC is a subset of Collaborators, it's sufficient to mention nodejs/collaborators only.

Reply:
@Trott I am fine with that, just thinking people might be using different email filters for different mentions.